lms/djangoapps/course_wiki/plugins/__init__.py

```diff
@@ -63,6 +63,5 @@ def get_allowed_operations(cls, course_key: CourseKey, user: Optional['User'] =
         return {
             # The wiki cannot be enabled/disabled via the API yet.
             "enable": False,
-            # There is nothing to configure for Wiki yet.
-            "configure": False,
+            "configure": True,
```

Comment:
Is it possible to configure wiki on a course basis? Where is the configuration stored in case of wiki?